fluid/PaddleCV/human_pose_estimation/train.py

```diff
@@ -30,18 +30,18 @@
 parser = argparse.ArgumentParser(description=__doc__)
 add_arg = functools.partial(add_arguments, argparser=parser)
 # yapf: disable
-add_arg('batch_size',       int,   32,                   "Minibatch size.")
-add_arg('dataset',          str,   'mpii',               "Dataset")
-add_arg('use_gpu',          bool,  True,                 "Whether to use GPU or not.")
-add_arg('num_epochs',       int,   140,                  "Number of epochs.")
-add_arg('total_images',     int,   144406,               "Training image number.")
-add_arg('kp_dim',           int,   16,                   "Class number.")
-add_arg('model_save_dir',   str,   "output",             "Model save directory")
-add_arg('with_mem_opt',     bool,  True,                 "Whether to use memory optimization or not.")
-add_arg('pretrained_model', str,   None,                 "Whether to use pretrained model.")
-add_arg('checkpoint',       str,   None,                 "Whether to resume checkpoint.")
-add_arg('lr',               float, 0.001,                "Set learning rate.")
-add_arg('lr_strategy',      str,   "piecewise_decay",    "Set the learning rate decay strategy.")
+add_arg('batch_size',       int,   128,                          "Minibatch size.")
```

Comment:
Minibatch size per GPU? or total? need to more clearly

Reply:
Done